README.md

```diff
@@ -58,7 +58,7 @@ The only extra JavaScript library we should load is [`Axios`](https://axios-http
 
 To download the `axios` node module, run `yarn install`
 
-To include axios in your project, include the following script tag below the script tag linking `index.js`:
+To include axios in your project, include the following script tag above the script tag linking `index.js`:
```

Comment:
Technically, it _shouldn't_ matter, so long as they don't try to call `axios` until after DOMContentLoaded has fired. From a "JS scripts load in order" standpoint, and since many students _do_ try to call `axios` outside an event handler while getting it up an running, it feels more natural to me to include axios _before_ the app script (as this change suggests).

So I like this change since it's easier to explain the reasoning (list dependencies before the thing that uses it), though again, it shouldn't be required.